roles/cmok.rb

```diff
@@ -0,0 +1,38 @@
+name "cmok"
+description "Master role applied to cmok"
+
+default_attributes(
+  :networking => {
+    :intefaces => {
+      :external => {
+        :interface => "ens3",
+        :role => :external,
+        :inet => {
+          :prefix => "24",
```

Comment:
We don't know yet what the internal address will be?

Reply:
No. It will be on the same network as wawel, but the 10.x IPs are not statically assigned.

Reply:
Well wawel's is being statically assigned now... I asked about whether it needed to be DHCP and was told now. If it needs to be DHCP then that will need some work in the cookbooks.

Reply:
OpenStack neuron assigns the internal IP for the life of the VM (or more specially the interface). There is no need for DHCP. If we change the interface or the emulated type (not that we would, virt-nic is best) then we would briefly need to switch back to DHCP.